tabs/tab_data.py

```
@@ -1,64 +1,120 @@
-from shiny import ui, module, reactive, render
-from shiny.types import FileInfo
-import pandas as pd
+import html as _html
+from typing import Any, Dict, List, Optional, Union, cast
```

Ruff (0.14.13)

`typing.Dict` is deprecated, use `dict` instead

(UP035)